### ISSUE TITLE
Monitor error when AddLand event is present before Estate event received

### ISSUE THREAD

**Issue by abarmat** [decentraland/marketplace#405]

**Comment by abarmat** [comment]
```[parcelReducer] [AddLand] Adding "-40,135" as part of the estate id "8"
(node:3376) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'asset_id' of undefined
    at _callee$ (/home/ubuntu/app/monitor/reducers/estateReducer.js:56:48)
    at tryCatch (/home/ubuntu/app/node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (/home/ubuntu/app/node_modules/babel-polyfill/node_modules/regenerator-runtime/runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (/home/ubuntu/app/node_modules/babel-polyfill/node_modules/regenerator-runtime/r
untime.js:117:21)
    at step (/home/ubuntu/app/monitor/reducers/estateReducer.js:204:191)
    at /home/ubuntu/app/monitor/reducers/estateReducer.js:204:361
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
(node:3376) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async fun
ction without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3376) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not hand
led will terminate the Node.js process with a non-zero exit code.
```

**Comment by nachomazzara** [comment]
Closed by #398
